Workbooks/SapMonitor/OsMetrics/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name": "OS Metrics",
```

Comment:
Need to confirm the workbook's name as this name will be displayed under "Monitoring" blade on the portal